.github/workflows/release.yml

```diff
@@ -100,3 +100,18 @@ jobs:
           ref: master
           token: ${{ secrets.NOIR_REPO_TOKEN }}
           inputs: '{ "noir-ref": "${{ needs.release-please.outputs.tag-name }}" }'
+
+  publish-wasm:
+    name: Publish noir_wasm package
+    needs: [release-please]
+    if: ${{ needs.release-please.outputs.tag-name }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dispatch to noir_wasm
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: update.yml
+          repo: noir-lang/noir_wasm
+          ref: master
+          token: ${{ secrets.NOIR_REPO_TOKEN }}
```

Comment:
We need to add this secret to that repository or generate a new one and change this.

Reply:
Made this secret available to `noir_wasm` as its the easiest path. Upto you guys if you want to change it to be separate secrets etc, when we do the devops handover